daffodil-runtime1/src/main/scala/org/apache/daffodil/udf/UDFService.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.udf
+
+import collection.JavaConverters._
+
+object UDFService {
```

Comment:
Ok, so UDFunctionService is written in Java, but UDFService object is in scala. So renaming the java class name to UDFService may not work well with the java/scala boundary, and it is confusing since this object is not the scala companion object for that Java class.

Suggest naming this UDFServiceFactory.

Reply:
Do we even need the scala version? Looks like it's just a very thin wrapper around the java version without providing much extra.